### DISCUSSION TITLE
Can you add the ability to filter out Wordle posts?

### DISCUSSION THREAD

**Discussion by Kwipper** [insin/control-panel-for-twitter#105] [Ideas]
First, I just want to say awesome add-on! Keep up the great work!

Can you add the ability to filter out Wordle posts? I see about 10 of these a day in my feed and it's really annoying to look at. Would be nice if I could toggle an option to filter these kinds of posts out.

**Comment by BD9Max** [comment]
Why dont you press that in the ... Menu that every tweet has?

![Screenshot_20220506-012009~2.jpg](https://user-images.githubusercontent.com/82726385/167234292-d6c64561-cd40-44af-a35b-e253383cf426.jpg)

I'd really much liked to see the three dot menus of  every tweet and user profiles better accessible - quick and fast - as you wrote that u get like 10 same tweets in your timeline.

You can add a filter to such as ublock origin to block those tweets that mentions your word.

But about that .... Menu - how much easier it would be if when those 3dots hovered a dropdown sub menu would appear fast no needless clicking.
Fancy CSS circle menus: https://freefrontend.com/css-circle-menus/


If u use ublock origin add this to "my filters":
twitter.com##article:has(a[href^="/i/user/"]:has-text(/wordle$/)):nth-ancestor(2)
